system-test/storage.js

```diff
@@ -1385,14 +1378,6 @@ describe('storage', function() {
           })
         );
 
-        it(
```

Comment:
I removed this because testing permissions is still allowed, even without providing a userProject on a requester pays bucket. There was either an API change that now allows this, or something else was busted on our end.